src/services/edusharing/index.js

```diff
@@ -27,6 +29,15 @@ class EduSharing {
 	}
 }
 
+class EduSharingPlayer {
+	get(uuid) {
+		if (!Configuration.get('ES_API_V7')) {
+			throw new MethodNotAllowed('This feature is disabled on this instance');
+		}
+		return EduSharingConnectorV7.getPlayerForNode(uuid);
```

Comment:
We want to avoid directly exports. The reason for it is coming from cleancode context.
const xxx = EduSharingConnectorV7.getPlayerForNode(uuid)
// empty line
return xxx;

Reply:
Done in last commit.